ext/pdo_odbc/odbc_driver.c

```diff
@@ -344,6 +344,30 @@ static bool odbc_handle_set_attr(pdo_dbh_t *dbh, zend_long attr, zval *val)
 			}
 			H->assume_utf8 = bval;
 			return true;
+		case PDO_ATTR_AUTOCOMMIT:
```

Comment:
Do you think it's also necessary to handle this constant in `odbc_handle_get_attr` ? I see some other implementations doing that.

Reply:
Thanks, I forgot about get_attr. I fixed it on https://github.com/php/php-src/pull/12780/commits/759e664243995d2784266188410a8f6dc61bb2e1